services/market/jobs/binance.js

```diff
@@ -13,21 +13,35 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const logger = require('lisk-service-framework').Logger();
+const {
+	Exceptions: { ServiceUnavailableException },
+	Logger,
+} = require('lisk-service-framework');
+
 const { reload } = require('../shared/market/sources/binance');
 
+const logger = Logger();
+
+const reloadMarketPrices = async () => reload()
+	.catch(err => {
+		if (err instanceof ServiceUnavailableException) {
+			logger.warn('Unable to fetch market prices from Binance right now. Will retry later.');
+		}
+		throw err;
+	});
```

Comment:
Kindly ensure that the control flow ends after `ServiceUnavailableException` is handled. Same for the other jobs.

```suggestion
const reloadMarketPrices = async () => reload()
	.catch(err => {
		if (err instanceof ServiceUnavailableException) {
			logger.warn('Unable to fetch market prices from Binance right now. Will retry later.');
			return;
		}
		throw err;
	});
```
or
```suggestion
const reloadMarketPrices = async () => reload()
	.catch(err => {
		if (err instanceof ServiceUnavailableException) {
			logger.warn('Unable to fetch market prices from Binance right now. Will retry later.');
		} else {
			throw err;
		}
	});
```